server/run-delivery.sh

```diff
@@ -31,7 +31,7 @@ OUTPUT_IMAGE=registry.centos.org/${APPID}/${TO}
 NS="${APPID}-${JOBID}-${DESIRED_TAG}"
 
 _ "Send success mail for user notify tube"
-python /tube_request/send_notify_request.py ${BEANSTALK_SERVER} ${OUTPUT_IMAGE} ${NOTIFY_EMAIL} ${TEST_TAG} ${NS} ${JOBID} ${LOGS_DIR}
+# python /tube_request/send_notify_request.py ${BEANSTALK_SERVER} ${OUTPUT_IMAGE} ${NOTIFY_EMAIL} ${TEST_TAG} ${NS} ${JOBID} ${LOGS_DIR}
```

Comment:
@bamachrn can you help answer this? I'm not entirely sure if I did the right thing here. :smile: